### ISSUE TITLE
@hulaizh share surfdrive data folder, make account, etc.

### ISSUE THREAD

**Issue by hannesdatta** [hannesdatta/tisem-airbnb#7]
Yes, SURFdrive is available via the WEbDAV protocol. However, an inherent aspect of the WebDAV protocol is that all actions are implemented via the server, which results in a slow user interface. This is why we recommend to always use the desktop clients.

If you want to make use of WebDAV, log in via https://surfdrive.surf.nl/files and then go to your personal page. To make use of WebDAV, you must create a 12-character password. This password will be generated for you and will be visible only once. Important: spaces are shown in the password to improve legibility. These spaces are not part of the password. Always enter the password directly in the application of your choice or save it in a good password manager. More information on the configuration of your WebDAV client you can find in the ownCloud documentation.
